### PR TITLE
Add changelog for 4.0.0-beta.4

### PR DESCRIPTION
**Pull request by consideRatio**

### CODE REVIEW

File: docs/source/changelog.md [hunk 1]
```diff
@@ -14,6 +14,14 @@ changes in pull requests], this list should be updated.
 
 ## 4.0
 
+### 4.0.0-beta.4 - 2024-10-11
+
+#### Maintenance and upkeep improvements
+
```
Comment (manics) [inline docs/source/changelog.md:20]:
```suggestion

-  Update ldapauthenticator from 1.3.2 to 2.0.0.b2 [#126](https://github.com/manics/zero-to-jupyterhub-k8s/pull/126)
```
If we get the ldapauthenticator beta in this means all major components changes will be in a Z2JH beta.

Comment (consideRatio) [reply docs/source/changelog.md:20]: Oooh, this was meant to be part of the beta.1 already, i messed up.

Comment (consideRatio) [reply docs/source/changelog.md:23]: Updated and pushed a commit to include this with a notice about this mistakenly being announced to be done in beta.1 but wasn't actually part of a release until now.

File: docs/source/changelog.md [hunk 2]
```diff
@@ -14,6 +14,27 @@ changes in pull requests], this list should be updated.
 
 ## 4.0
 
+### 4.0.0-beta.4 - 2024-10-11
+
+ldapauthenticator was said to be bumped to version from version 1.3.2 to 2.0.0b2
```
Comment (manics) [inline docs/source/changelog.md:19]:
```suggestion
ldapauthenticator was said to be bumped from version 1.3.2 to 2.0.0b2
```

**Pull request merged**